gax/src/main/java/com/google/api/gax/core/PropertiesProvider.java

```diff
@@ -54,6 +54,7 @@ public static String loadProperty(Class<?> loadedClass, String propertiesPath, S
       if (inputStream != null) {
         Properties properties = new Properties();
         properties.load(inputStream);
+        inputStream.close();
```

Comment:
IIUC `close()` itself can throw an IO exception so we should put it in a `try` block.